app/controllers/katello/api/v2/repositories_controller.rb

```diff
@@ -151,6 +153,8 @@ def create
       repository.verify_ssl_on_sync = ::Foreman::Cast.to_bool(repo_params[:verify_ssl_on_sync]) if repo_params.key?(:verify_ssl_on_sync)
       repository.upstream_username = repo_params[:upstream_username] if repo_params.key?(:upstream_username)
       repository.upstream_password = repo_params[:upstream_password] if repo_params.key?(:upstream_password)
+      repository.ostree_upstream_sync_policy = repo_params[:ostree_upstream_sync_policy] if repo_params.key?(:ostree_upstream_sync_policy)
+      repository.ostree_upstream_sync_depth = repo_params[:ostree_upstream_sync_depth] if repo_params.key?(:ostree_upstream_sync_depth)
```

Comment:
nitpick  You could use   repo_params.fetch(:ostree_upstream_sync_policy, nil)    here for these two lines